frontend/packages/dev-console/src/components/import/validation-schema.ts

```diff
@@ -148,8 +148,8 @@ export const limitsValidationSchema = yup.object().shape({
           const { requestUnit, limit, limitUnit } = this.parent;
           if (limit !== undefined) {
             return (
-              convertToBaseValue(`${request}${requestUnit}`) <=
-              convertToBaseValue(`${limit}${limitUnit}`)
+              (requestUnit ? convertToBaseValue(`${request}${requestUnit}`) : request) <=
+              (limitUnit ? convertToBaseValue(`${limit}${limitUnit}`) : limit)
```

Comment:
So `convertToBaseValue` returns the number only, if unit is not present. That means it should work here as when cores is selected the selectedKey is an empty string and hence the respective unit value should be an '' ". Instead of adding an extra check here which is already present in  `convertToBaseValue` you should try to figure out why we are getting undefined instead of " " here. Units should never be undefined because we are setting a default unit and in case it changes we should have that unit value or " " in case of cores. If you figure out this is some yup issue then we can add this change for the time being.

Reply:
I have removed these changes and added `ensure` in the yup validation of `requestUnit` and `limitUnit`